addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -452,6 +461,116 @@ QUnit.test('do not post message with "Enter" keyboard shortcut', async function
     );
 });
 
+QUnit.test("Display suggested recipients list", async function (assert) {
+    assert.expect(10);
+
+    this.data['res.partner'].records.push({ id: 100 });
+    await this.start({
+        async mockRPC(route, args) {
+            if (route === '/mail/get_suggested_recipients') {
+                return Promise.resolve(
+                    {
+                        100: [
+                            [
+                                false,
+                                "john@example.com",
+                                'CC email',
+                            ],
+                            [
+                                false,
+                                "fred@example.com",
+                                'CC email',
+                            ],
+                            [
+                                100,
+                                "second partner",
+                                "CC email"
+                            ],
+                            [
+                                false,
+                                "carl@example.com",
+                                'CC email',
+                            ],
+                        ]
+                    });
+            }
+            return this._super(...arguments);
+        },
```

Comment:
Provide `data` so that default mocked RPC is correct, instead of mocking this route explicitly in test with hard-coded result...

... Maybe the complication is on suggested recipient that is not linked to a partner... TBH, I wouldn't mind having a fake mock server model just for that, e.g. `mail.suggested_recipient_without_partner`. I know it's different from the server code, but it's much safer that way that hard-coding the result of this RPC.